src/main/java/step3/domain/LottoNo.java

```diff
@@ -0,0 +1,52 @@
+package step3.domain;
+
+import java.util.Objects;
+
+public class LottoNo implements Comparable {
```

Comment:
Comparable의 타입을 명확히 해주는건 어떨까요?:)
```suggestion
public class LottoNo implements Comparable<LottoNo> {
```
이렇게 되면 compareTo()도 바꿔야하긴 합니다:)
제네릭 타입을 사용할땐 Raw 타입을 그대로 사용하지 않도록 주의해야 합니다!
아래 블로그를 참고해주세요!
https://madplay.github.io/post/dont-use-raw-types

Reply:
그리고 LottoNumber로 풀네임을 사용하는건 어떨까요? :)
https://myeonguni.tistory.com/1596 **9. 2 Classes** 를 참고해주세요!

> 완전한 단어를 사용하고 두 문자어와 약어는 피하도록 한다(만약, 약어가 URL이나 HTML과 같이 더 많이 ,더 넓게 사용되고 있다면 약어를 사용하는 것도 괜찮다).

Reply:
앗.. 이거 다음 스텝 요구사항이 LottoNo 로 구현하는거라서 이렇게 했습니다!